packages/api/src/external/fhir/document/index.ts

```diff
@@ -76,7 +78,51 @@ export function getBestDateFromCWDocRef(content: DocumentContent): string {
   return date.toISOString();
 }
 
-export const toFHIR = (
+export const cqToFHIR = (
+  docId: string,
+  doc: IHEGWDocumentReference,
+  patientId: string,
+  fhirDocRef?: DocumentReference
+): DocumentReferenceWithId => {
+  const baseAttachment = {
+    ...(doc.fileName ? { fileName: doc.fileName } : {}),
+    ...(doc.contentType ? { contentType: doc.contentType } : {}),
+    ...(doc.size ? { size: doc.size } : {}),
+    ...(doc.creation ? { creation: doc.creation } : {}),
+  };
+
+  const metriportFHIRContent = fhirDocRef
+    ? createDocReferenceContent({
+        ...baseAttachment,
+        location: doc.url ?? "",
```

Comment:
unless I'm missing something, it seems like we're using optional values a bit too liberally throughout - we should never have an empty location here, right?